js/views/pageNavVw.js

```diff
@@ -91,17 +96,48 @@ module.exports = Backbone.View.extend({
   },
 
   handleSocketMessage: function(response) {
-    var data = JSON.parse(response.data);
-    if(data.id == this.socketNotificationID){
-      console.log(data);
-    }
-  },
+    var data = JSON.parse(response.data),
+        username,
+        avatar,
+        n;
+
+    if (data.hasOwnProperty('notification')) {
+      n = data.notification;
+      username = n.handle ? n.handle : n.guid.substring(0,10) + '...';
+      avatar = n.image_hash ? app.serverConfig.getServerBaseUrl + '/get_image?hash=' +
+        n.image_hash + '&guid=' + n.guid : 'imgs/defaultUser.png';
+
```

Comment:
Since you're truncating the username here, it might be a good idea to add it to the notification model and use it in the notification list, then we could get rid of the duplicate logic there that checks if a handle is available and shows a guid truncated by css if it isn't.